en-GB/Setup.js

```diff
@@ -55,20 +55,20 @@ export default {
     'AttemptingToConnect': 'Attempting to detect emulator... ',
     'RunEmulatorAndOpenGame': 'Please run your emulator & Open the game',
     'CurrentlySupportedEmulators': '(Only DeSmuME, VBA & Citra supported currently)',
-    'CheckForX64': 'Check your emulator is "x64" and not "x86" or ask in discord',
+    'CheckForX64': 'Check your emulator is "x64" and not "x86" or ask in Discord',
     'MultipleEmulatorsDetected': 'Multiple emulators detected',
     'NoEmulatorsFound': 'No emulators found, click to try again?',
     'EmulatorFound': 'Emulator Detected! (using {emulator})',
     'SelectADetectedEmulatorLong': 'Please select one of the emulators we found running:',
     'SelectADetectedEmulatorShort': 'Select an Emulator',
-    'DoesNotSupportGeneration': '{emulator} (Doesn\'t support gen {generation})',
+    'DoesNotSupportGeneration': '{emulator} (Doesn\'t support Gen {generation})',
     'DetectionIsPaused': 'Emulator detection paused',
     'WaitingForGameData': 'Waiting for Game data...',
     'ConnectingToEmulator': 'Attempting to automatically connect to {emulator}',
     'LoadLuaScriptIntoEmulator': 'Load the Lua Script into your emulator',
     'Instructions': {
-      'VBA-RR': 'This can be done by going to <em>Tools</em> » <em>Lua Script Window</em> » <em>New Lua Script Window</em> inside VBA-ReRecording <em>(Be sure not to close the script window, you can minimize it though)</em',
-      'DeSmuME': 'This can be done by going to <em>Tools</em> » <em>Lua Scripting</em> » <em>New Lua Script Window</em> inside of <em>DeSmuME</strong> <em>(Be sure not to close the script window, you can minimize it though)</em>'
+      'VBA-RR': 'This can be done by going to <em>Tools</em> » <em>Lua Script Window</em> » <em>New Lua Script Window</em> inside <em>VBA-ReRecording</em> <em>(Be sure not to close the script window, you can minimize it though)</em',
+      'DeSmuME': 'This can be done by going to <em>Tools</em> » <em>Lua Scripting</em> » <em>New Lua Script Window</em> inside of <em>DeSmuME</em> <em>(Be sure not to close the script window, you can minimize it though)</em>'
```

Comment:
i don't know how these `<em>`-Tags are used later on, but wouldn't one opening and closing Tag be enough there? like `<em>DeSmuME (Be sure not to close the script window, you can minimize it though)</em>`

Reply:
same for `VBA-ReRecording` as well

Reply:
They are for styling (they make bold text in HTML), my guess is that those get rendered in the app.

About incorporating the last part in a single tag, I understand but I am not super sure.
It does the same, however it logically becomes a full block.

Except this single point, I agree and have no issue changing that. 🙂

Reply:
Having tags in language strings isn't perfect, I know. They are going to be removed at a later point.